src/main/java/org/opensearch/security/dlic/rest/api/RestApiAdminPrivilegesEvaluator.java

```diff
@@ -98,13 +99,6 @@ public boolean isCurrentUserAdminFor(final Endpoint endpoint, final String actio
             return false;
         }
         if (adminDNs.isAdmin(userAndRemoteAddress.getLeft())) {
-            if (logger.isDebugEnabled()) {
-                logger.debug(
-                    "Security admin permissions required for endpoint {} but {} is not an admin",
-                    endpoint,
-                    userAndRemoteAddress.getLeft().getName()
-                );
-            }
```

Comment:
I thought this was wrong so I removed it

Reply:
It was originally added for debugging purposes.

Reply:
Got it. What I meant is that the logic is inverted, the logging should state that the user is an admin.